src/validation/validate_request.py

```
@@ -0,0 +1,43 @@
+import json
+from pathlib import Path
```

Flake8 (7.2.0)

[error] 'pathlib.Path' imported but unused

(F401)